project/plugins.sbt

```diff
@@ -16,6 +16,7 @@ addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.2.24")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.3")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.1.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.20.0")
```

Comment:
I read this readme and still don't understand it.  Do we need npm now?

Reply:
scalajs bundler works with either npm and yarn, and we are using it with yarn. Unfortunately we do need it for Scala.js support, for [hpack.js](https://www.npmjs.com/package/hpack.js). Hopefully at some point we can replace this with a pure Scala HPACK implementation e.g. https://github.com/davenverse/hpack.